drivers/include/sps30.h

```diff
@@ -95,11 +95,11 @@ typedef struct {
     float mc_pm2_5;     /**< Mass concentration of PM 2.5 [µg/m^3] */
     float mc_pm4;       /**< Mass concentration of PM 4.0 [µg/m^3] */
     float mc_pm10;      /**< Mass concentration of PM 10 [µg/m^3] */
-    float nc_pm0_5;     /**< Number concentration of PM 0.5 [µg/m^3] */
-    float nc_pm1;       /**< Number concentration of PM 1.0 [µg/m^3] */
-    float nc_pm2_5;     /**< Number concentration of PM 2.5 [µg/m^3] */
-    float nc_pm4;       /**< Number concentration of PM 4.0 [µg/m^3] */
-    float nc_pm10;      /**< Number concentration of PM 10 [µg/m^3] */
+    float nc_pm0_5;     /**< Number concentration of PM 0.5 [#/cm^3] */
+    float nc_pm1;       /**< Number concentration of PM 1.0 [#/cm^3] */
+    float nc_pm2_5;     /**< Number concentration of PM 2.5 [#/cm^3] */
+    float nc_pm4;       /**< Number concentration of PM 4.0 [#/cm^3] */
+    float nc_pm10;      /**< Number concentration of PM 10 [#/cm^3] */
```

Comment:
`#` is not very clear - why not spell it out?

> particles / cm³

Reply:
I simply used to follow the convention in the sps30_saul file. The comments in that file mentions `#/m^3`.
Also, I think PM1.0, PM2.5, PM4.0, PM10 are  a pretty standard representation of particle sizes, so I didn't think one would need to specifically mention that

Reply:
If you like `#/cm³` better, I won't object that.
But "Number concentration of PM 1.0" just makes me scratch my head and then it gets all sore and the numbers in these registers go up…

Reply:
Ohk yeah thats a valid point. Will change this